src/PrestaShopBundle/Resources/config/services/adapter/alias.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.adapter.alias.deleter:
+    class: 'PrestaShop\PrestaShop\Adapter\Search\AliasDeleter'
```

Comment:
as mentioned bellow, you can delete this service and don't bother about deletion in this PR.